codec_specs.md

```diff
@@ -642,6 +640,14 @@ Description: DTS Lossless audio that does not have a core substream. The private
 
 Initialization: none
 
+### A_EAC3
+
+Codec ID: A_EAC3
+
+Codec Name: Dolby Digital Plus / DD+ / Enhanced AC-3 / E-AC-3
```

Comment:
```suggestion
Codec Name: Dolby Digital Plus / E-AC-3
```

For coherency, use `Dolby Digital / AC-3` and `Dolby Digital Plus / E-AC-3` everywhere.